sources/minishell.h

```diff
@@ -22,6 +22,14 @@
 # include "resources.h"
 # include "commands/commands.h"
 
+# include "tokenizing/input_processing.h"
+# include "tree_make/tree_processing.h"
+# define FREE_ENVVARS_DLISTS 1
```

Comment:
where did you get these? I removed them long time ago

Reply:
don't really know.

Reply:
Plz remove the defines then